clang/lib/Basic/OpenMPKinds.cpp

```diff
@@ -760,6 +760,9 @@ void clang::getOpenMPCaptureRegions(
     case OMPD_parallel:
       CaptureRegions.push_back(OMPD_parallel);
       break;
+    case OMPD_assume:
+      CaptureRegions.push_back(OMPD_unknown);
+      break;
```

Comment:
Please don't push OMPD_unknown here, instead add OMPD_assume to the directives at lines 790+.

Reply:
Also, please add OMPD_assume to `isOpenMPCapturingDirective`.

Reply:
Why? We usually add OMPD_unknown to the directives that do nit require outlining to a standalone function, here should be the same behavior.

Reply:
This is what happens with the directives at lines 790+ (before these changes).  We don't want to be adding OMPD_unknown as we go, because it can end up intermingled with other directives (in case of combined directives).  I understand that OMPD_assumes may be standalone, but we should treat it the same as the combinable ones for consistency.

Reply:
It may affect codegen + may add some memory overhead

Reply:
The final output from this (global) function will be the same, so it shouldn't affect codegen.